mesos/src/cgcloud/mesos/mesos_box.py

```diff
@@ -69,7 +69,7 @@ def mesos_service( name, *flags ):
                            '$(cat /var/lib/mesos/slave_args)' ) ] )
 
 
-class MesosBoxSupport( GenericUbuntuTrustyBox, Python27UpdateUbuntuBox, CoreMesosBox ):
+class MesosBoxSupport( GenericUbuntuDefaultBox, Python27UpdateUbuntuBox, CoreMesosBox ):
```

Comment:
I think I understand the motivation for this but it seems brittle. This change introduces the illusion of the ability to effortlessly upgrade concrete box classes to a new Ubuntu release, while the reality looks different. Typically, bumping the Ubuntu release for a box requires a number of other tweaks in subclasses.